plugins/flytekit-greatexpectations/flytekitplugins/great_expectations/task.py

```diff
@@ -140,7 +140,7 @@ def _flyte_schema(self, dataset) -> str:
                 dataset.remote_path, self._local_file_path, is_multipart=True
             )
 
-        # DataFrame (Pandas, Spark, etc.)
+        # DataFrame (pandas, Spark, etc.)
```

Comment:
how about 'S'park?

Reply:
The [documentation](https://spark.apache.org/docs/latest/) uses uppercase 'S'.

Reply:
This looks odd, @SmritiSatyanV. Let's revert it or have `pandas, spark`.